packages/components/src/components/rating-stars/rating-stars.css

```diff
@@ -10,81 +10,151 @@
  */
 
 :host {
-  --rating-spacing: 0.125rem;
-}
+  --scl-stars-color: var(--scl-color-primary);
+  --scl-stars-hover-color: var(--scl-color-primary-hover);
+  --scl-stars-active-color: var(--scl-color-primary-active);
+  --scl-stars-placeholder-color: var(--scl-color-black);
+  --scl-stars-placeholder-revert-color: var(--scl-color-black);
+  --scl-stars-disabled-color: var(--scl-color-grey-50);
+  --scl-stars-spacing: var(--scl-spacing-2);
 
-.rating__label {
-  position: absolute;
-  bottom: var(--scl-spacing-32, 2rem);
+  display: inline-flex;
+  flex-direction: column;
 }
 
-.rating--disabled .rating__label {
-  color: var(--scl-color-grey-50, #7c7c7c);
+[part='wrapper']:hover .icon-clip {
+  --decimal-width: 100% !important;
```

Comment:
is this `--decimal-width` actually doing anything? since the `precision` prop is gone but maybe there's something I'm missing… 🤔

Reply:
you are right, this one was no longer needed ;-)